svc-bip-api/src/main/java/gov/va/vro/bip/config/BipApiConfig.java

```diff
@@ -22,13 +21,12 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.security.KeyManagementException;
-import java.security.KeyStore;
-import java.security.KeyStoreException;
-import java.security.NoSuchAlgorithmException;
+import java.security.*;
```

Comment:
Do we want to use wildcard imports?

Reply:
yea not sure why it was a wildcard, IDE might have done it auto.